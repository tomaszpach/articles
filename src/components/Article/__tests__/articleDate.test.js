import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import ArticleDate from "../articleDate";

describe("ArticleDate", () => {
  test("should correctly render ArticleDate component", () => {
    const wrapper = shallow(<ArticleDate date="5. februar 2019" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
