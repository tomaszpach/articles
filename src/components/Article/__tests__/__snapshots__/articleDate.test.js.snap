// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleDate should correctly render ArticleDate component 1`] = `
<span
  className="article-date"
>
  5. februar 2019
</span>
`;
